dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -104,8 +104,12 @@ RUN python3 -m pip install aiohttp \
                  scapy==2.5.0 \
                  setuptools-rust \
                  six \
-                 snappi==1.27.1 \
-                 snappi-ixnetwork==1.27.2 \
+                 snappi==1.33.0 \
+                 snappi-ixnetwork==1.33.1 \
+                 matplotlib \
+                 seaborn \
+                 rich \
+                 influxdb_client \
```

Comment:
we don't need influxdb_client. we will use open telemetry lib instead. i will create the PR for that later. the telemetry pipeline PR is close to be ready.

Reply:
fixed